useraccount.py

```
@@ -4,7 +4,7 @@
 
 home_dir = os.getenv("HOME")
 # ---
-if home_dir:
+if os.getenv("HOME"):
     project = "/data/project/himo"
 else:
     project = 'I:/core/bots/core1'
```

Ruff (0.11.9)

Use ternary operator `project = "/data/project/himo" if os.getenv("HOME") else 'I:/core/bots/core1'` instead of `if`-`else`-block

Replace `if`-`else`-block with `project = "/data/project/himo" if os.getenv("HOME") else 'I:/core/bots/core1'`

(SIM108)